domain-models-runtime/src/main/java/org/folio/rest/tools/utils/ValidationHelper.java

```diff
@@ -41,85 +36,76 @@ public static Errors createValidationErrorMessage(String field, String value, St
     return e;
   }
 
-  public static void handleError(Throwable error, Handler<AsyncResult<Response>> asyncResultHandler){
-
-    if(error != null){
+  public static void handleError(Throwable error, Handler<AsyncResult<Response>> asyncResultHandler) {
+    if (error != null) {
       Response r = null;
-      Map<Character,String> errorMessageMap = PgExceptionUtil.getBadRequestFields(error);
-      if(errorMessageMap != null){
+      Map<Character, String> errorMessageMap = PgExceptionUtil.getBadRequestFields(error);
+      if (errorMessageMap != null) {
         //db error
         String desc = errorMessageMap.getOrDefault('D', "");
         String mess = errorMessageMap.getOrDefault('M', "");
-        try{
-          if(isInvalidUUID(mess)){
+        try {
+          if (isInvalidUUID(mess)) {
             int start = mess.indexOf('\"');
-            int end = mess.indexOf('\"', start+1);
-            String value = "";
-            if(start != -1 && end != -1 && start < end){
-              value = mess.substring(start+1, end);
+            int end = mess.indexOf('\"', start + 1);
+            String UUIDvalue = "";
+            if (start != -1 && end != -1 && start < end) {
+              UUIDvalue = mess.substring(start + 1, end);
             }
-            r = withJsonUnprocessableEntity(ValidationHelper.createValidationErrorMessage("", value, mess));
-          }
-          else if(isDuplicate(mess) || isFKViolation(mess)){
+            r = withJsonUnprocessableEntity(ValidationHelper.createValidationErrorMessage("", UUIDvalue, mess));
+          } else if (isDuplicate(mess) || isFKViolation(mess)) {
             String[] errorDesc = desc.split("=");
-            String field = errorDesc[0].substring(errorDesc[0].indexOf('(')+1,errorDesc[0].indexOf(')'));
-            String value = errorDesc[1].substring(errorDesc[1].indexOf('(')+1,errorDesc[1].indexOf(')'));
+            String field = errorDesc[0].substring(errorDesc[0].indexOf('(') + 1, errorDesc[0].indexOf(')'));
+            String value = errorDesc[1].substring(errorDesc[1].indexOf('(') + 1, errorDesc[1].indexOf(')'));
+            if (isDuplicate(mess))
+              mess = "duplicate " + field + " value violates unique constraint :" + value;
```

Comment:
It's normally colon followed by blank (and not the other way around)
```suggestion
              mess = "duplicate " + field + " value violates unique constraint: " + value;
```